saltychat/SaltyClient/VoiceManager.cs

```diff
@@ -548,6 +548,17 @@ private void OnMessage(dynamic message, dynamic cb)
                         )
                     );
                 }
+            }
+            
+             if (pluginState.IsReady)
+            {
+                BaseScript.TriggerEvent(Event.SaltyChat_ConnectedToTeamspeak, pluginState.isReady);
+
+            }
+            else
+            {
+                BaseScript.TriggerEvent(Event.SaltyChat_ConnectedToTeamspeak, pluginState.isReady);
+
             }
```

Comment:
This if/else makes no sense at all, you check a condition and execute the exact same code...
Also there is no check for a state change, so every time there is a `StateUpdate` (e.g. if you talk) this just fires the event.
Implement this in the if-block a few lines above.

Reply:
When i tested it using a debug message each time it didnt respond to talking. 
Its meant to be there so u could get the information if a player is connected and in the right channel from say another script and not just internal in saltychat.